installing/installing_ibm_z/upi/installing-ibm-z-kvm.adoc

```diff
@@ -102,7 +95,7 @@ include::modules/cluster-telemetry.adoc[leveloffset=+1]
 [role="_additional-resources"]
 .Additional resources
 
-* See xref:../../../support/remote_health_monitoring/about-remote-health-monitoring.adoc#about-remote-health-monitoring[About remote health monitoring] for more information about the Telemetry service
+* xref:../../../support/remote_health_monitoring/about-remote-health-monitoring.adoc#about-remote-health-monitoring[About remote health monitoring]
```

Comment:
```suggestion
* xref:../../../support/remote_health_monitoring/about-remote-health-monitoring.adoc#about-remote-health-monitoring[About remote health monitoring].
```
There's a period on the next link and this is a complete sentence, so I would add a period here (or remove the next).

Reply:
remove both - there should only be the text of the link title in the `[]` and nothing outside per guidelines